### PR TITLE
Move all of the testloader files to a single page so it's easier to l…

### LINKED ISSUE #41: Make it easier to load up sample data
We have to go to a bunch of URL's over and over to get the sample data in dev, it should be pretty straightforward to have a simpler process for now. (That can be iterated and improved later)

### PR DESCRIPTION
**Pull request by jonespm**
…oad the

test data. Fixes #41

### CODE REVIEW

**Comment by jonespm** [comment]
Just looks like this for now. Just a quick hack that can be quickly run after re building the server/db.

image

File: dashboard/urls.py [hunk 1]
```diff
@@ -45,7 +47,9 @@
     url(r'^update_groups', cron.update_groups, name='update_groups'),
     url(r'^submission', cron.submission, name='submission'),
     url(r'^weight_consideration', cron.weight_consideration, name='weight_consideration'),
-
+    url(r'^testloader', TemplateView.as_view(
```
Comment (zqian) [inline dashboard/urls.py]: This is good. I need to move other templates here following the TemplateView format. Nice work!

Comment (jonespm) [reply dashboard/urls.py]: Yeah this pattern seemed pretty easy to just display templates, we can re-organize the other templates on a future PR.

Comment (jonespm) [reply dashboard/urls.py]: I think there's some other refactoring that would be nice to do too, like breaking up views.py a bit before it gets too large.

**Pull request merged**